src/TreeView/TreeView.tsx

```diff
@@ -169,19 +169,24 @@ const Item: React.FC<TreeViewItemProps> = ({
             }
           }}
           sx={{
+            '--toggle-width': '1rem', // 16px
             position: 'relative',
             display: 'grid',
-            gridTemplateColumns: `calc(${level - 1} * 8px) 16px 1fr`,
+            gridTemplateColumns: `calc(${level - 1} * (var(--toggle-width) / 2)) var(--toggle-width) 1fr`,
             gridTemplateAreas: `"spacer toggle content"`,
             width: '100%',
-            height: 32,
+            height: '2rem', // 32px
             fontSize: 1,
             color: 'fg.default',
             borderRadius: 2,
             cursor: 'pointer',
             '&:hover': {
               backgroundColor: 'actionListItem.default.hoverBg'
             },
+            '@media (pointer: coarse)': {
+              '--toggle-width': '1.5rem', // 24px
```

Comment:
Nice! Is there a specific value for min touch target size in the guideline? Because I increased the touch target size on the UnderlineNav as well but used the value from design. I am curious to see if there is a guideline

Reply:
Not sure. @mperrotti might know!